bundles/model/org.eclipse.smarthome.model.core/src/main/java/org/eclipse/smarthome/model/core/internal/folder/FolderObserver.java

```diff
@@ -238,6 +238,9 @@ private static void checkFile(final ModelRepository modelRepo, final File file,
                         if (parsers.contains(getExtension(file.getName()))) {
                             try (FileInputStream inputStream = FileUtils.openInputStream(file)) {
                                 modelRepo.addOrRefreshModel(file.getName(), inputStream);
+                            } catch (Exception e) {
```

Comment:
I rather go with your last option. I didn't want to change the original error to warn, but handle this specific case differently. That's why I added the catch to the existing "inner try".
Your suggestion with catching an IOException  at the outer catch makes sense to me as it keeps the original error message and just prints warnings in the case of an IO error.